com.somaxa8.earx.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.somaxa8.earx</id>
+  
+  <name>EarX</name>
+  <summary>A simple application to control your Nothing earbuds ANC (Active Noise Cancellation) modes.</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>AGPL-3.0-only</project_license>
+  
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+  
+  <description>
+    <p>
+      A simple Rust application using gtk-rs and Adw (Libadwaita) to control your Nothing earbuds ANC (Active Noise Cancellation) modes. This application demonstrates how to build a native desktop interface with Rust, async/await (via Tokio), and Bluetooth functionalities (via bluer). 
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">com.somaxa8.earx.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>screenshot.png</image>
```

Comment:
this is invalid. It should be a URL that accessible like previously, but pointing to the actual image and not the github page.